cli/lib/utils.ts

```diff
@@ -50,15 +52,18 @@ const RULE_AUTOGENERATED_COMMENT = `
   ${padEnd('//', 80, '/')}
 `
 
-/** Generate the role script text */
-export async function generateRuleScript(ruleDef: RuleDefinition) {
+/** Generate the code of the Auth0 script as a string */
+export async function generateScript(
```

Comment:
I might write an overload for this to ensure that you only allow `RuleDefinitions` when `scriptType === 'rules'` and vice versa.

Reply:
Did you not see my comment about that in the PR description?

> You previously recommended that I declare a function overload for generateScript, using the scriptType to determine the type of the manifest definitions. The parts of the interfaces we use, however, are fully overlapping, so I didn't bother.